scripts/install_idyntree_bindings.m

```diff
@@ -0,0 +1,118 @@
+% creates an installation error later (see
+% https://github.com/robotology/robotology-superbuild/issues/780)
+if ~ispc
+    if contains(strip(install_prefix), ' ')
+        fprintf('Install directory path %s contains a space.\n', install_prefix)
+        fprintf('Please use install_idyntree_bindings function in a directory path that does not have spaces.\n');
+        return;
+    end
+end
+
+setup_script = fullfile(pwd, 'idyntree_bindings_setup.m');
+
+if exist(install_prefix)
+    fprintf('Directory %s already present.\n', install_prefix);
+    fprintf('Please use the existing installation or delete the existing installation to proceed with the install.\n');
+    return;
+end
+
+fprintf('Installing iDynTree Matlab bindings in %s\n', install_prefix);
+
+% The install url is created following
+mambaforge_url_prefix = 'https://github.com/conda-forge/miniforge/releases/latest/download/';
+if ispc
+    mambaforge_installer_name = 'Mambaforge-Windows-x86_64.exe';
+elseif ismac
+    [~, uname_m] = system('uname -m');
+    % Remove newline
+    uname_m = strip(uname_m);
+    mambaforge_installer_name = sprintf('Mambaforge-MacOSX-%s.sh', uname_m);
+elseif isunix
+    [~, uname] = system('uname');
+    % Remove newline
+    uname = strip(uname);
+    [~, uname_m] = system('uname -m');
+    % Remove newline
+    uname_m = strip(uname_m);
+    mambaforge_installer_name = sprintf('Mambaforge-%s-%s.sh', uname, uname_m);
+end
+
+fprintf('Downloading mambaforge installer \n');
+mambaforge_installer_url = strcat(mambaforge_url_prefix, mambaforge_installer_name);
+websave(mambaforge_installer_name, mambaforge_installer_url);
+fprintf('Download of mambaforge installer completed\n');
+
+% See https://github.com/conda-forge/miniforge#non-interactive-install
+fprintf('Installing mambaforge\n');
+if ispc
+    system(sprintf('start /wait "" %s /InstallationType=JustMe /RegisterPython=0 /S /D=%s', mambaforge_installer_name, install_prefix));
+    conda_full_path = fullfile(install_prefix, 'condabin', 'conda.bat');
+    % On Windows, the files in conda are installed in the Library
+    % subdirectory of the prefix
+    idyntree_install_prefix = fullfile(install_prefix, 'Library');
+elseif isunix
+    system(sprintf('sh %s -b -p "%s"', mambaforge_installer_name, install_prefix));
+    conda_full_path = fullfile(install_prefix, 'bin', 'conda');
+    idyntree_install_prefix = install_prefix;
+end
+fprintf('Installation of mambaforge completed\n');
+
+if ~exist(install_prefix, 'dir')
+    fprintf('Installation in %s failed for unknown reason.\n', install_prefix);
+    fprintf('Please open an issue at https://github.com/ami-iit/paper_ramadoss-2022-ral-humanoid-base-estimation/issues/new .\n');
+    return;
+end
+
+% Install all the idyntree-matlab-bindings
+fprintf('Installing idyntree-matlab-bindings from robotology\n');
+system(sprintf('"%s" install -y -c conda-forge -c robotology idyntree-matlab-bindings', conda_full_path));
```

Comment:
Also in this case, better to include the exact version of as many packages you used, to ensure that the same version will be used by the users. Furthermore this ensure that the solver is able to solve the environment faster:
```suggestion
system(sprintf('"%s" install -y -c conda-forge -c robotology idyntree-matlab-bindings=5.0.0 idyntree=5.0.1', conda_full_path));
```

Reply:
Thank you. Changed in https://github.com/ami-iit/paper_ramadoss-2022-ral-humanoid-base-estimation/pull/1/commits/2295787f336f3825bc6366453dbd792b52198801